### ISSUE TITLE
Default display size is too large

### ISSUE THREAD

**Issue by subsoap** [indiesoftby/defold-scene3d#1]
1920x1080 is large. It would be better to halve what's in the game.project file so it's 960x540.

Reasoning for this is that not all users have large screens and the default display size can cause issues for them if they are running on a PC.

If a larger initial view size is desired it could be set with DefOS based on user configs.

**Comment by aglitchman** [comment]
I think this is "a tooling problem" if Defold can't set the meaningful size of the window on the game start.

Why 1920x1080? It is the current most popular screen resolution -> it's a good idea to develop games that will look great on 1920x1080 screens (btw, in our projects, we use 2560x1440). 960x540 will require using upscaled graphics and scaling everything x0.5 to get great-looking graphics on FHD displays. I think it's counterproductive.

So, I decided to keep the resolution 1920x1080, plus I added the code https://github.com/indiesoftby/defold-scene3d/commit/231f8cf5399a34ef075f470dfc1c66988d9ac2cc that fits the window in the centre of the current display.
Let's see how it goes.
